src/Components/InstanceTypes/SuggestedInstanceTypesTable.js

```diff
@@ -0,0 +1,100 @@
+    { instanceType: 't2.large', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 4 },
+    { instanceType: 'm2.2xlarge', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 },
+    { instanceType: 'm3.2xsmall', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 6 },
+    { instanceType: 'm3.2xlarge', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 }
+];
+
+export default function SuggestedInstanceTypesTable() {
+
+    const [page, setPage] = useState(PAGE);
+    const [perPage, setPerPage] = useState(PER_PAGE);
+
+    const onSetPage = (event, page) => setPage(page);
+    const onPerPageSelect = (event, perPage) => {
+        setPerPage(perPage);
+        setPage(PAGE);
+    };
+
+    return (
+        <>
+            <section className='pf-l-page__main-section pf-c-page__main-section'>
+                <Card>
+                    <CardBody>
+                        <PrimaryToolbar
+                            pagination={{
+                                itemCount: 50,
+                                page,
+                                perPage,
+                                onSetPage,
+                                onPerPageSelect
+                            }}
+                        />
+                        <TableComposable
+                            aria-label='suggested instance types table'
+                            variant={TableVariant.compact}
+
+                        >
+                            <Thead noWrap>
+                                <Tr>
+                                    {
+                                        SUGG_INSTANCE_TYPES_TABLE_COLUMNS.map(column => {
+                                            return <Th key={column}>{column}</Th>;
+                                        })
+                                    }
+                                </Tr>
+                            </Thead>
+                            <Tbody>
+                                {
+                                    SUGG_INSTANCE_TYPES_DATA.map(instance => {
+                                        return (
+                                            <Tr key={instance.instanceType}>
+                                                <Td>{instance.instanceType}</Td>
+                                                <Td>{instance.provider}</Td>
+                                                <Td>{instance.description}</Td>
+                                                <Td>{instance.count}</Td>
+                                            </Tr>
+                                        );
+                                    })
+                                }
+                            </Tbody>
+                        </TableComposable>
+                        <Pagination
```

Comment:
@r14chandra I would suggest to add `widgetId` to identify elements on the page in case of testing.